bedrock/firefox/templates/firefox/profile/profile-base.html

```diff
@@ -0,0 +1,82 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros.html" import fxa_email_form with context %}
+
+{% add_lang_files "firefox/profile-per-install" %}
```

Comment:
@peiying2 Strings are split across three different templates but two of them just have one small bit of text, so I think all three should share a single lang file. Will this work?

Reply:
Let me check and get back to you.